assignments/callbacks.js

```diff
@@ -26,26 +26,59 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
-  
+  return cb(items.length);
 }
+getLength(items,function(length){
+  console.log(length);
+});
+
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  for(let i=0;i<items.length;i++){
+    return cb(items[3]);
+  }
+  
 }
+last(items,function(item){
+  console.log(item);
+});
+
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return x+y;
```

Comment:
You have to pass in `x+y` in `cb`. Your solution looks kind of works because when `sumNums` was invoked, it actually returned `x+y`. 

This code 
```js
function(add){
  console.log(add);
}
```
was not used at all. To verify, try invoking the anonymous function you passed in to `sumNums` by adding `()`:
```js
sumNums(2,3,function(add){
  console.log(add);
}());
```
It will log `undefined`.